draft-ietf-quic-transport.md

```diff
@@ -2203,6 +2203,21 @@ address before path validation is complete.
 A client that migrates to a new address SHOULD use a preferred address from the
 same address family for the server.
 
+## Use of IPv6 Flow-Label and Migration {#ipv6-flow-label}
+
+Endpoints that send data using IPv6 SHOULD apply an IPv6 flow label
```

Comment:
```suggestion
Endpoints that send data using IPv6 SHOULD set an IPv6 flow label for a QUIC connection
```

Reply:
It is actually per path, not per connection, and is explained in the next paragraph.
@martinthomson suggested replacing "set" by "apply" in a previous comment.